### PR TITLE
Providing an example of button ui component

### PR DESCRIPTION
**Pull request by eduard13**
## Purpose of this pull request

This pull request (PR) provides an example of how to use a button ui component within a listing component.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/ui_comp_guide/components/ui-button.html
- https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-button.html

## Links to Magento source code
- https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/ReleaseNotification/view/adminhtml/ui_component/release_notification.xml#L304

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by rogyar** [comment]
From a technical perspective, this PR is correct. The link to an example from the core is attached to the description.

File: guides/v2.2/ui_comp_guide/components/ui-button.md [hunk 1]
```diff
@@ -29,6 +29,40 @@ Option | Description | Type | Required |
 `actionName` | Name of the component's method to be invoked. | String | Required |
 `params` | A list of arguments that will be passed to the method. | Array | Optional |
 
+## Examples
+
+### Configure component
+
+Here is an example of how Button component integrates with Listing component:
```
Comment (bdenham) [inline guides/v2.2/ui_comp_guide/components/ui-button.md]:
```suggestion
Here is an example of a Button component ("Test Button") added below a Listing component:
```

File: guides/v2.2/ui_comp_guide/components/ui-button.md [hunk 2]
```diff
@@ -29,6 +29,40 @@ Option | Description | Type | Required |
...
+        <displayAsLink>false</displayAsLink>
+        <title><![CDATA[Test Button]]></title>
+    </settings>
+</button>
+...
+```
+
+### Result
+
+The result is a new added button to page listing.
```
Comment (bdenham) [inline guides/v2.2/ui_comp_guide/components/ui-button.md]:
```suggestion
The Button component appears below a Listing component on the page, as follows:
```

**Comment by eduard13** [comment]
Hi @bdenham, the suggestions have been applied.
Thank you.

**Comment by bdenham** [comment]
running tests

**Pull request merged**